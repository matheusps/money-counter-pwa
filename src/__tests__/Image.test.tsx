import React from 'react'
import { render } from 'react-testing-library'
import 'jest-styled-components'

import Image from '../components/Image'

describe('Image component', () => {
  const component = render(<Image src="testing" />)

  it('should render', () => {
    expect(component).toBeTruthy()
  })

  it('should match snapshot', () => {
    expect(component.asFragment()).toMatchSnapshot()
  })
})
