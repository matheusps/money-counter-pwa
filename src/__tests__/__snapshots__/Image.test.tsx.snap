// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image component should match snapshot 1`] = `
<DocumentFragment>
  <img
    class=""
    src="testing"
  />
</DocumentFragment>
`;
